### PR TITLE
Bugfix: payment gateway code cleanup

### PR DESCRIPTION
**Pull request by defunctl**
- Remove returns of void functions.
- Add `@return never` docBlocks, as wp_send_json_* will always execute `die` at the end.
- Update code formatting.

### CODE REVIEW

File: plugins/wme-sitebuilder/wme-sitebuilder/Wizards/PaymentGatewayPayPal.php [hunk 1]
```diff
@@ -67,69 +67,77 @@ public function props() {
...
 		if ( ! $response->wasSuccessful() ) {
-			wp_send_json_error(new WP_Error(
+			wp_send_json_error( new WP_Error(
 				'mapps-wpcli-error',
 				sprintf(
-					/* Translators: %1$s is the exit code from WP CLI; %2$s is output from WP CLI. */
+				/* Translators: %1$s is the exit code from WP CLI; %2$s is output from WP CLI. */
 					__( 'Encountered WP CLI exit code "%1$s" with output "%2$s".', 'wme-sitebuilder' ),
 					sanitize_text_field( $response->getExitCode() ),
 					sanitize_text_field( $response->getOutput() )
```
Comment (tarecord) [inline plugins/wme-sitebuilder/wme-sitebuilder/Wizards/PaymentGatewayPayPal.php:106]: ❓ Is comment indentation correct here?

File: plugins/wme-sitebuilder/wme-sitebuilder/Wizards/PaymentGatewayStripe.php [hunk 2]
```diff
@@ -67,81 +67,91 @@ public function props() {
...
 		if ( ! $response->wasSuccessful() ) {
-			wp_send_json_error(new WP_Error(
+			wp_send_json_error( new WP_Error(
 				'mapps-wpcli-error',
 				sprintf(
-					/* Translators: %1$s is the exit code from WP CLI; %2$s is output from WP CLI. */
+				/* Translators: %1$s is the exit code from WP CLI; %2$s is output from WP CLI. */
 					__( 'Encountered WP CLI exit code "%1$s" with output "%2$s".', 'wme-sitebuilder' ),
 					sanitize_text_field( $response->getExitCode() ),
 					sanitize_text_field( $response->getOutput() )
```
Comment (tarecord) [inline plugins/wme-sitebuilder/wme-sitebuilder/Wizards/PaymentGatewayStripe.php:106]: ❓ Indentation again?

**Review by tarecord — APPROVED** [review]
Two minor things, otherwise, looks good to me!

**Review by aykanburcak — APPROVED** [review]

**Review by bswatson — APPROVED** [review]

**Comment by defunctl** [comment]
Closed in favor of https://github.com/moderntribe/wme/pull/204

**Pull request closed without merging**